### PR TITLE
Ble ctlr without mpsl

### PR DESCRIPTION
**Pull request by sofu1**
Just for running CI.

### CODE REVIEW

**Comment by sofu1** [comment]
Draft PR to make the final PR go fast.
This PR is for having ble_controller and MPSL as two separate libraries, i.e. that ble_controller does not contain MPSL but is linked together with it in NCS.
The clock driver (previously clock_control_nrfxlib.c) now only uses MPSL API, so I renamed it and moved it out of bt_ll_nrfxlib.
@rugeGerritsen @thomasstenersen @joerchan @minkargar @frederik-vestre-nordic @anangl

File: lib/multithreading_lock/multithreading_lock.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
 /**
  * @file multithreading_lock.h
  *
- * @brief APIs for ensuring BLE controller threadsafe operation.
+ * @brief APIs for ensuring MPSL threadsafe operation.
```
Comment (sofu1) [inline lib/multithreading_lock/multithreading_lock.h]: MPSL and ble_controller

Comment (sofu1) [reply lib/multithreading_lock/multithreading_lock.h]: or remove both

File: subsys/CMakeLists.txt [hunk 2]
```diff
@@ -1,11 +1,13 @@
 #
-# Copyright (c) 2018 Nordic Semiconductor ASA
+# Copyright (c) 2018 - 2019 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
 if (CONFIG_BT)
   add_subdirectory(bluetooth)
+elseif(CONFIG_MPSL)
```
Comment (sofu1) [inline subsys/CMakeLists.txt]: should be if

File: subsys/bluetooth/controller/hci_driver.c [hunk 3]
```diff
@@ -308,6 +309,7 @@ static void signal_thread(void *p1, void *p2, void *p3)
 
 	while (true) {
 		k_sem_take(&sem_signal, K_FOREVER);
+		/* Call mpsl_low_priority_process(); when DRGN-12631 is done */
 		ble_controller_low_prio_tasks_process();
```
Comment (sofu1) [inline subsys/bluetooth/controller/hci_driver.c]: this will be removed

File: subsys/bluetooth/controller/hci_driver.c [hunk 4]
```diff
@@ -368,53 +370,79 @@ u8_t bt_read_static_addr(bt_addr_le_t *addr)
 	return 0;
 }
 
-static int ble_init(struct device *unused)
+static u8_t m_config_clock_source_get(void)
```
Comment (sofu1) [inline subsys/bluetooth/controller/hci_driver.c]: will be in mpsl subsys instead

File: subsys/bluetooth/controller/hci_driver.c [hunk 5]
```diff
@@ -565,7 +593,6 @@ static int hci_driver_init(struct device *unused)
 	IRQ_CONNECT(SWI5_IRQn, BLE_CONTROLLER_IRQ_PRIO_LOW,
```
Comment (sofu1) [inline subsys/bluetooth/controller/hci_driver.c]: remove

**Comment by joerchan** [comment]
What? Is this a duplicate?

**Comment by sofu1** [comment]
> What? Is this a duplicate?

Yes, it was just a draft that I made earlier for the actual PR to go faster

**Pull request closed without merging**